stellar_core_commander.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "faraday", "~> 0.9.1"
   spec.add_dependency "faraday_middleware", "~> 0.9.1"
   spec.add_dependency "pg", "~> 0.18.1"
+  spec.add_dependency "sqlite3", "~> 1.3.13"
```

Comment:
I don't think we need the sqlite3 package as you're not actually using it from commander (it is used/installed by stellar-core but that is taken care of by the stellar-core packaging solution)